code/game/objects/structures/tables_racks.dm

```diff
@@ -23,6 +23,7 @@
 	pass_flags_self = PASSTABLE | LETPASSTHROW
 	layer = TABLE_LAYER
 	obj_flags = CAN_BE_HIT | IGNORE_DENSITY
+	var/table_climbable = TRUE
```

Comment:
I don't understand why there would be any confusion with the actual climbable datum. Are you storing that, or is that an existing variable somewhere higher up? If it's already used that's fine but it should be just climbable otherwise.

Also document (`/// Explanation here` before the line)

Reply:
It's not an existing var anywhere higher up on that object, I just wanted it to have a unique name, but I can change it

Reply:
done